crates/wasmtime/src/runtime.rs

```diff
@@ -818,6 +819,7 @@ pub(crate) struct StoreInner {
     host_info: RefCell<HashMap<HostInfoKey, Rc<RefCell<dyn Any>>>>,
     externref_activations_table: Rc<VMExternRefActivationsTable>,
     stack_map_registry: Rc<StackMapRegistry>,
+    null_func_ref: Box<VMCallerCheckedAnyfunc>,
```

Comment:
Ah it looks like there is indeed a null funcref per-store, but perhaps this could be reused for each `Instance`?